scholia/app/static/scholia.js

```diff
@@ -539,3 +539,419 @@ function sparqlToShortInchiKey(sparql, key,  element, filename) {
+	        q  + 
+	       '&format=json&callback=?';
+     
+    $.getJSON(url, function (data) {
+	 var item = data.entities[q];
+	 if ('en' in item.labels) {
+	     $("#h1").text(item.labels.en.value);
+	 }
+	 $("#h1").append(' (<a href="https://www.wikidata.org/wiki/' +  q  + ' ">' + q  + '</a>)');
+	 
+	 var detailsList = Array();
+	 try {
+	     var orcid = item.claims.P496[0].mainsnak.datavalue.value;
+	     detailsList.push( '<a href="https://orcid.org/"><img alt="' +
+			       'ORCID logo" src="{{ url_for(\'static\', filename=\'images/orcid_16x16.gif\') }}"' +
+			       'width="16" height="16" hspace="4" /></a> <a href="https://orcid.org/' +
+			       encodeURI( orcid ) + '">https://orcid.org/' + escapeHTML( orcid ) + '</a>');
+	 }
+	 catch(e) {}
+
+	 try {s
+	     var mastodonAccount = item.claims.P4033[0].mainsnak.datavalue.value;
+	     var mastodonComponents = mastodonAccount.split("@");
+	     if (mastodonComponents.length == 2) {
+		 detailsList.push( '<img alt="Twitter t icon" src="' +
+				   'https://upload.wikimedia.org/wikipedia/commons/4/48/Mastodon_Logotype_%28Simple%29.svg' +
+				   '" width="16" height="16" hspace="4" /> <a href="https://' + mastodonComponents[1] + '/@' +
+				   encodeURI( mastodonComponents[0] ) + '">@' + escapeHTML( mastodonComponents[0] ) + '</a> ');
+	     }
+	 }
+	 catch(e) { }
+
+	 try {
+	     var twitterAccount = item.claims.P2002[0].mainsnak.datavalue.value;
+	     detailsList.push( '<img alt="Twitter t icon" src="' +
+			       'https://upload.wikimedia.org/wikipedia/commons/8/8b/Twitter_logo_initial.svg' +
+			       '" width="16" height="16" hspace="4" /> <a href="https://twitter.com/' +
+			       encodeURI( twitterAccount ) + '">@' + escapeHTML( twitterAccount ) + '</a> ');
+	 }
+	 catch(e) { }
+
+	 try {
+	     var rorID = item.claims.P6782[0].mainsnak.datavalue.value;
+	     detailsList.push( 'ROR <a href="https://ror.org/' + encodeURI( rorID ) + '">' + rorID + '</a> ');
+	 }
+	 catch(e) { }
+
+	 try {
+	     var hashtag = item.claims.P2572[0].mainsnak.datavalue.value;
+	     detailsList.push( '<a href="https://hashtags-hub.toolforge.org/' +
+			       encodeURI( hashtag ) + '">#' + escapeHTML( hashtag ) + '</a> ');
+	 }
+	 catch(e) {}
+
+	 try {
+	     var doi = item.claims.P356[0].mainsnak.datavalue.value;
+	     $("head").append( '<meta name="citation_doi" content="' + doi + '"/>' );
+	 }
+	 catch(e) {}
+
+	 try {
+	     $( '#details' ).append( detailsList.join( " | " ) );
+	 }
+	 catch(e) {}
+	 
+	 /* BioSchemas annotation */
+	 if (item.claims.P31 &&
+	     ((item.claims.P31[0].mainsnak.datavalue.value.id == 'Q5'))) {
+	   try { /* Person */
+	       bioschemasAnnotation = {
+	          "@context" : "https://schema.org",
+	          "@type" : "Person" ,
+	          "http://purl.org/dc/terms/conformsTo": { "@type": "CreativeWork", "@id": "https://bioschemas.org/profiles/Person/0.2-DRAFT-2019_07_19/" },
+	          "description" : "A person" ,
+	          "identifier" : "{{ q }}" ,
```

Comment:
For example, this line can be changed to:
```js
"identifier" : q,
```
as `q` is passed as a parameter to this function